hexrd/ui/calibration/auto/powder_runner.py

```diff
@@ -222,14 +222,11 @@ def update_config(self):
             #        this array is the reduce lattice parameter set.
             refined_lattice_params = self.ic.full_params[self.ic.npi:]
             self.material.latticeParameters = refined_lattice_params
+            HexrdConfig().material_modified(self.material)
```

Comment:
I think this should be `self.material.name` instead.

Reply:
Oh, and we need the `emit()` too. It should also be `HexrdConfig().material_modified.emit(self.material.name)`

Reply:
Both fixed now